champ/node/rpc/account.rs

```diff
@@ -28,4 +27,18 @@ impl Account for AccountService {
             None => Err(Status::new(tonic::Code::Internal, "missing Block data")),
         }
     }
+    async fn get_next_block_height (&self, request: Request<NextBlockHeightRequest>) -> Result<Response<NextBlockHeightReply>, Status> {
```

Comment:
Please run rust fmt, some of the spacing etc is off. You can also change the max line length to something like 100

https://github.com/rust-lang/rustfmt/blob/master/Configurations.md